external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -80,10 +80,12 @@ bool isElementwiseOp(Operation *op) {
         tosa::NegateOp,
```

Comment:
One of my lost earlier versions had this function as "isFuseableOp" instead of "isElementwiseOp," since they aren't all element-wise any more.